easybuild/tools/testing.py

```diff
@@ -264,6 +264,11 @@ def post_easyconfigs_pr_test_report(pr_nr, test_report, msg, init_session_state,
         'os_version': system_info['os_version'],
         'pyver': system_info['python_version'].split(' ')[0],
     }
+
+    # also mention CPU architecture name, but only if it's known
+    if system_info['cpu_arch_name'] != UNKNOWN:
+        short_system_info['cpu_model'] += " (%s)" % system_info['cpu_arch_name']
```

Comment:
`short_system_info` is a string, not a dictionary

Reply:
nice catch, thanks...

fixed in 8a1115f